Tests/Functional/Asynchronous/Expectations/main.swift

```diff
@@ -551,7 +551,37 @@ class ExpectationsTestCase: XCTestCase {
         RunLoop.main.run(until: Date() + 1)
     }
 
-    static var allTests = {
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsAsync' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsAsync' failed \(\d+\.\d+ seconds\)
+    func test_waitForExpectationsAsync() async {
+      // Basic check that waitForExpectations() is functional when used with the
+      // await keyword in an async function.
+      let expectation = self.expectation(description: "foo")
+      expectation.fulfill()
+      await self.waitForExpectations(timeout: 0.0)
+    }
+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor' failed \(\d+\.\d+ seconds\)
+    @MainActor func test_waitForExpectationsFromMainActor() {
+      // Basic check that waitForExpectations() is functional and does not need
+      // the await keyword when used from a main-actor-isolated test function.
+      let expectation = self.expectation(description: "foo")
+      expectation.fulfill()
+      self.waitForExpectations(timeout: 0.0)
+    }
+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor_async' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Case 'ExpectationsTestCase.test_waitForExpectationsFromMainActor_async' failed \(\d+\.\d+ seconds\)
+    @MainActor func test_waitForExpectationsFromMainActor_async() async {
+      // Basic check that waitForExpectations() is functional and does not need
+      // the await keyword when used from a main-actor-isolated test function.
+      let expectation = self.expectation(description: "foo")
+      expectation.fulfill()
+      self.waitForExpectations(timeout: 0.0)
```

Comment:
nit: these new tests appear to be using 2-space indent

Reply:
Fixed.